pkg/status/status.go

```diff
@@ -178,6 +180,8 @@ func (s *status) ensureClusterOperator() error {
 			Namespace: s.namespace,
 		},
 	})
+	s.setRelatedObjects()
```

Comment:
There's a comment [here](https://github.com/operator-framework/operator-marketplace/pull/205/files#diff-a5666326aefdf43a8e89fd4871449fa0R260) that might be a good idea to include above this particular line too for consistency.

Reply:
Reason to add a comment should be to explain something that is not obvious and not for consistency. The reason for the comment in the other place is to convey to the user why we are always updating `RelatedObjects` even though it is static.

Reply:
I don't believe that `setRelatedObjects` needs to be called given that it is called in the `updateStatus` function. Am I missing a usecase?

Reply:
What if `must-gather` is executed in the window before the `updateStatus()` call?

Reply:
Plus this the place where we are creating the object on a fresh cluster, so it should have all the required fields and not depend on it being done elsewhere.

Reply:
> What if must-gather is executed in the window before the updateStatus() call?
The call to the client to create the object on cluster occurs before the  `setRelatedObjects()` function so this would still be an issue.

> Plus this the place where we are creating the object on a fresh cluster, so it should have all the required fields and not depend on it being done elsewhere.

I believe that we should always ensure that required static fields like this one are set in update calls.

Reply:
>The call to the client to create the object on cluster occurs before the setRelatedObjects() function so this would still be an issue.

Good point. I will fix that.

>I believe that we should always ensure that required static fields like this one are set in update calls.

And I am doing that. But we should talk more about this when you are doing the refactor.

Reply:
>And I am doing that. But we should talk more about this when you are doing the refactor.

Agreed!